src/goose/defaults.clj

```diff
@@ -11,12 +11,16 @@
 (def ^:no-doc in-progress-queue-prefix "goose/in-progress-jobs:")
 (def ^:no-doc process-prefix "goose/processes:")
 (def ^:no-doc heartbeat-prefix "goose/heartbeat:")
+(def ^:no-doc batch-prefix "goose/batch:")
 
 (def default-queue "default")
 (def schedule-queue "scheduled-jobs")
 (def dead-queue "dead-jobs")
 (def cron-queue "cron-schedules")
 (def cron-entries "cron-entries")
+(def enqueued-job-set "enqueued")
+(def successful-job-set "successful")
+(def dead-job-set "dead")
```

Comment:
suggestion: we should also capture callback function symbol for a batch

Reply:
As discussed, we will not have this in the `defaults` namespace, since it is specific to the Redis implementation of the batch state.